frontend/src/views/Pipelines/PipelineEnvironmentEditForm.js

```diff
@@ -29,28 +28,46 @@ import listEnvironmentGroups from '../../api/Environment/listEnvironmentGroups';
 import * as Defaults from '../../components/defaults';
 
 const PipelineEnvironmentEditForm = (props) => {
-  const { environmentOptions, triggerEnvSubmit, pipelineUri, pipeline, handleCountEnvironmentValid } = props;
+  const {
+    environmentOptions,
+    triggerEnvSubmit,
+    pipelineUri,
+    pipeline,
+    handleCountEnvironmentValid
+  } = props;
   const dispatch = useDispatch();
-  const { enqueueSnackbar } = useSnackbar();
   const client = useClient();
   const [kvEnvs, setKeyValueEnvs] = useState([]);
   const [envsToRemove, setEnvsToRemove] = useState([]);
   const [environments, setEnvironments] = useState([]);
-  const [mapGroups, setMapGroups] = useState(new Map())
-  const stageOps =[{value:"dev", label:"dev"},{value:"test", label:"test"},{value:"val", label:"val"},{value:"prod", label:"prod"},{value:"other", label:"other"}];
-  const [environmentOps, setEnvironmentOps] = useState(
-    environmentOptions && environmentOptions.length > 0 ? environmentOptions : [{ environmentUri: 'someUri', label: 'some' },{ environmentUri: 'someUri', label: 'some2' }]
-  );
+  const [mapGroups, setMapGroups] = useState(new Map());
+  const stageOps = [
+    { value: 'dev', label: 'dev' },
+    { value: 'test', label: 'test' },
+    { value: 'val', label: 'val' },
+    { value: 'prod', label: 'prod' },
+    { value: 'other', label: 'other' }
+  ];
+
+  const environmentOps =
+    environmentOptions && environmentOptions.length > 0
+      ? environmentOptions
+      : [
+          { environmentUri: 'someUri', label: 'some' },
+          { environmentUri: 'someUri', label: 'some2' }
+        ];
 
   useEffect(() => {
     if (client && pipeline) {
-      console.log("useeffect")
-      console.log(pipeline)
-      const environmentsSorted = pipeline.developmentEnvironments.nodes.sort((a, b) => {
-        return a.order - b.order;
-      });
+      console.log('useeffect');
```

Comment:
Same here

Reply:
Thank you for the 2 comments, I added a rule to check for the logs and removed debugging logs